outgoing.go

```diff
@@ -157,7 +157,11 @@ func readVersionFile() {
 func main() {
 	flag.Parse()
 	if *secretKey == "" {
-		log.Fatal("-key must be set.")
+		if os.Getenv("OUTGOING_SECRET_KEY") == "" {
+			log.Fatal("-key must be set.")
+		} else {
```

Comment:
Not a big deal, but this else can be removed since the program will exit on line 161.